### DISCUSSION TITLE
How do imodel files relate to database information?

### DISCUSSION THREAD

**Discussion by jiangpeiqi** [iTwin/itwinjs-core#418] [Q&A]
Is the Imodel Bridge a similar protocol for matching model ids to database ids?

**Comment by kabentley** [comment]
I'm sorry, but you're going to have to be more specific about what you are asking. iModel bridges (now called Connectors) convert data from a source format into an iModel. Since each source format is different, each Connector works differently for creating things like model names, model hierarchies, etc. 

What type of Ids are you asking about?
